components/sidebar/topLevelNavItems.jsx

```diff
@@ -33,7 +33,7 @@ var MENU_ENTRIES = [
         to: '/clubs'
       },
       {
-        name: "Guides & Resources",
+        name: "guides_and_resources",
```

Comment:
in the interested of dev sense, should these become `locale_id` rather than `name``?